### ISSUE TITLE
Unhandled Info.plist files in SPM

### ISSUE THREAD

**Issue by katebertelsen** [Iterable/swift-sdk#415]
We have recently updated our project from using Carthage to using SPM for managing our dependencies.

We started with the Iterable SDK at version 6.2.10, because that's what we used under Carthage. Today, we updated to 6.2.13 and we got the following warning, under `Resolve Package Graph`:

```
⚠️  found 1 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
[DerivedDataRedacted]/SourcePackages/checkouts/swift-sdk/notification-extension/Info.plist
⚠️ found 1 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
[DerivedDataRedacted]/SourcePackages/checkouts/swift-sdk/swift-sdk/Info.plist
```

I went back and checked each version between 6.2.10 and 6.2.13 inclusive and it appears that this was introduced with 6.2.12.

We like to run a zero-warning project if at all possible. What steps can I take on my side to resolve these warnings, outside of downgrading back to 6.2.11?

**Comment by roninopf** [comment]
Thanks for the report @katebertelsen ! This definitely sounds like we made a mistake on our end as we of course don't want to introduce warnings due to the SDK. We'll look into this ASAP. Thanks for checking on the versioning, as well.

I'm not aware of any fixes to this on your side, I think what I can best recommend is that you see the version changelog for each of those and judge which you can go with or without, until we resolve this. If there is a fix to this that doesn't require us to do something, I'd love to see it for reference.

**Comment by roninopf** [comment]
Hi @katebertelsen, if you'd like to try out the fix to your report before it's officially merged in, please feel free to point the SPM to the `MOB-2280-exclude-plists` branch! I tested the fix with a newly created sample project, but extra eyes and accountability is appreciated.

**Comment by katebertelsen** [comment]
It looks like that branch gets rid of the warnings on our side, too! Thanks so much!

**Comment by roninopf** [comment]
Great, thank you for the confirmation! We'll get this merged and released ASAP, and I'll update here when it's out.

**Comment by roninopf** [comment]
@katebertelsen We've released 6.2.15 which has the fix for this. Thanks again for the report and fix verification! If you have another problem please feel free to open a new issue here or with your CSM.
